### PR TITLE
Added mag_h0 from DSCOVR

### LINKED ISSUE #517: Add DSCOVR data import

### PR DESCRIPTION
**Pull request by abotiamnot**
fixes #517

### CODE REVIEW

**Comment by abotiamnot** [comment]
Hmm, that's weird.

**Comment by abotiamnot** [comment]
@dstansby Can you take a look at the AppVeyor Build?

File: heliopy/data/dscovr.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import astropy.units as u
+import datetime as dt
+from collections import OrderedDict
+
+from heliopy.data import util
+from heliopy import config
+
+data_dir = path.Path(config['download_dir'])
+dscovr_dir = data_dir / 'dscovr'
+dscovr_url = 'https://spdf.gsfc.nasa.gov/pub/data/dscovr/'
```
Comment (dstansby) [inline heliopy/data/dscovr.py]: 👍 👍 👍  I didn't know that DSCOVR data was on SPDF!

Comment (abotiamnot) [reply heliopy/data/dscovr.py]: Here's a really cool list we can use for adding more data sources! https://omniweb.sci.gsfc.nasa.gov/ftpbrowser/

**Comment by dstansby** [comment]
This is great, I've been trying to get DSCOVR data for ages.

**Pull request merged**